CONTRIBUTING.md

```diff
@@ -6,6 +6,13 @@ before you begin your work.
 
 ## Adding New Components
 
+**Before** any code is written, [open an issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/new?assignees=&labels=new+component&template=new_component.md&title=New%20component) providing the following information:
+
+* Who's the sponsor for your component. A sponsor is an approver who will be in charge of being the official reviewer of the code and become a code owner for the component. For vendor-specific components, it's good to have a volunteer sponsor. If you can't find one, we'll assign one in a round-robin fashion. For non-vendor specific components, having a sponsor means that your use case has been validated.
+* Some information about your component, such as the reasoning behind it, use-cases, telemetry data types supported, and anything else you think it's relevant for us to make a decision about accepting the component
```

Comment:
> for us to make a decision about accepting the component

Once the issue is opened, what is the process for making a decision? How is the decision indicated?

Reply:
What are your thoughts? The best I can think of is approvers/maintainers commenting on the issue that a decision has been made to approve/reject a proposal.

Reply:
Labels? "Proposed Component", "Approved Component", and after X weeks close the issue as no decision made => no interest in supporting this?

Reply:
> Labels? "Proposed Component", "Approved Component", and after X weeks close the issue as no decision made => no interest in supporting this?

I like this. The "Sponsor Needed" label can be applied until a decision is made.